servers/services/src/main/java/org/projectnessie/services/authz/Check.java

```diff
@@ -42,6 +42,7 @@
     CREATE_REFERENCE(true, false),
     ASSIGN_REFERENCE_TO_HASH(true, false),
     DELETE_REFERENCE(true, false),
+    DELETE_DEFAULT_BRANCH(false, false),
```

Comment:
due to its relation to `DELETE_REFERENCE` i put this right after (and did the same in all other files).
if we want (or need due to relying on enumeration number somewhere) new values to be added at the end, i can adjust them accordingly.

Reply:
That's fine here - we neither store nor use this enum's ordinal.